### PR TITLE
AST-2013 - Transparent Header Issue with Full Width/Stretched Layout

### PR DESCRIPTION
**Pull request by GauravKhupse**
### Description

The default margin of padding added to the Full Width/Stretched Layout is causing the issues with the Transparent header.

Steps To Reproduce: 

Install Astra on a Fresh Installation
Create a page with Full Width/Stretched Layout and add a full-Width block
Disable the title and enable the transparent header
Due to the padding, there is white space behind the transparent header and obviously, it does not look good.
It only happens with the Full Width/Stretched layout and the issue is fixed with Full Width/Contained.

### Screenshots

https://d.pr/i/hFYPPs

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: inc/dynamic-css/container-layouts.php [hunk 1]
```diff
@@ -115,7 +116,7 @@ function astra_container_layout_css() {
 				}
 			';
 			/** @psalm-suppress InvalidArgument */ // phpcs:ignore Generic.Commenting.DocComment.MissingShort
-			if ( true === apply_filters( 'astra_stretched_layout_with_spacing', true ) && false === astra_check_any_page_builder_is_active( astra_get_post_id() ) ) {
+			if ( true === apply_filters( 'astra_stretched_layout_with_spacing', true ) && false === astra_check_any_page_builder_is_active( astra_get_post_id() ) && false === $enable_trans_header ) {
```
Comment (imnavanath) [inline inc/dynamic-css/container-layouts.php:119]:
```suggestion
			if ( true === apply_filters( 'astra_stretched_layout_with_spacing', true ) && false === $enable_trans_header && false === astra_check_any_page_builder_is_active( astra_get_post_id() ) ) {
```

Comment (imnavanath) [reply inc/dynamic-css/container-layouts.php:119]: Keep function call at last as this needs to go through certain conditions.

**Comment by imnavanath** [comment]
@GauravKhupse please add changelog point.

File: inc/dynamic-css/container-layouts.php [hunk 2]
```diff
@@ -134,7 +135,7 @@ function astra_container_layout_css() {
 				}
 			';
 			/** @psalm-suppress InvalidArgument */ // phpcs:ignore Generic.Commenting.DocComment.MissingShort
-			if ( true === apply_filters( 'astra_stretched_layout_with_spacing', true ) && false === astra_check_any_page_builder_is_active( astra_get_post_id() ) ) {
+			if ( true === apply_filters( 'astra_stretched_layout_with_spacing', true ) && false === $enable_trans_header && false === astra_check_any_page_builder_is_active( astra_get_post_id() ) ) {
```
Comment (imnavanath) [inline inc/dynamic-css/container-layouts.php:138]: Instead only astra_get_option do we need to use theme's get transparent header function which checks meta option too?

Comment (GauravKhupse) [reply inc/dynamic-css/container-layouts.php]: I updated condition to check with title disable

File: inc/dynamic-css/container-layouts.php [hunk 3]
```diff
@@ -80,7 +80,8 @@ function astra_container_layout_css() {
 	$page_container_css        = '';
 	$customizer_default_update = astra_check_is_structural_setup();
 	$page_title_header_padding = ( true === $customizer_default_update ) ? '2em' : '4em';
-
+	// Transparent Header.
+	$display_title = get_post_meta( get_the_ID(), 'site-post-title', true );
```
Comment (imnavanath) [inline inc/dynamic-css/container-layouts.php:84]:
```suggestion
	$display_title = get_post_meta( (int) astra_get_post_id(), 'site-post-title', true );
```

**Review by imnavanath — APPROVED** [review]

**Pull request merged**